webpack/assets/javascripts/react_app/stores/NotificationsStore.test.js

```diff
@@ -18,11 +18,29 @@ describe('NotificationsStore', () => {
   });
 
   it('sorts groups by date descending', () => {
-    const result = NotificationsStore.prepareNotifications(data);
+    const result = NotificationsStore.prepareNotifications(NotificationsData);
 
     expect(result.test[0].id).toBe(6);
     expect(result.test[1].id).toBe(3);
     expect(result.info[0].id).toBe(18);
     expect(result.info[1].id).toBe(9);
   });
+
+  it('test when there are no notifications', () => {
+    const result = NotificationsStore.prepareNotifications([]);
+
+    expect(Object.keys(result).length).toBe(0);
+  });
+
+  it('returns false when there are no unread notifications', () => {
+    const result = NotificationsStore.hasUnreadNotifications(NotificationsData);
+
+    expect(result).toBeFalsy();
+  });
+
+  it('returns true when there are unread notifications', () => {
+    const result = NotificationsStore.hasUnreadNotifications(NotificationsDataSeen);
+
+    expect(result).toBeTruthy();
+  });
 });
```

Comment:
Both test data sets contain unread notifications, notification where property seen is false
method is not returning correct result